{"name": "test_get_all_pullrequests", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 > 0\n +  where 0 = len([])", "trace": "@pytest.mark.run(order=5)\n    def test_get_all_pullrequests():\n        api = Apirequests(read_base_url())\n        response = api.send_get(pulls, auth=auth)\n        get_prs_response = Apiresponses(response)\n        if get_prs_response.get_status_code() != 200:\n            print(f\"Failed to get PRs from repository: {get_prs_response.get_status_code()}\")\n    \n        pull_requests = json.loads(get_prs_response.get_content())\n        print(pull_requests)\n    \n>       assert len(pull_requests) > 0\nE       assert 0 > 0\nE        +  where 0 = len([])\n\ntest_get_all_pullrequests.py:25: AssertionError"}, "attachments": [{"name": "stdout", "source": "3810fec7-933c-44cf-9a21-c5d3e0ac66b8-attachment.txt", "type": "text/plain"}], "start": 1679160155225, "stop": 1679160155521, "uuid": "fc16ac3d-cf45-45c4-8d21-96a55c75d8f6", "historyId": "fbc6c2432ccea861bb7e931da3d7a713", "testCaseId": "7ddec853ec9e9dff91b025d66da117c6", "fullName": "test_get_all_pullrequests#test_get_all_pullrequests", "labels": [{"name": "tag", "value": "run(order=5)"}, {"name": "suite", "value": "test_get_all_pullrequests"}, {"name": "host", "value": "AlphaTest"}, {"name": "thread", "value": "7836-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_get_all_pullrequests"}]}